src/main/pages/user-management/user-management.adoc

```diff
@@ -59,16 +87,16 @@ Setting it to `embedded` will make it use Che’s OAuth Authenticator, for which
 Read more about link:ide_projects.html#project-import-and-ssh-connection[SHH key management].
 
 [id="protocol-based-providers"]
-== Protocol Based Providers
+== Protocol based providers
 
-Keycloak provides support for http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/saml.html[SAML v2.0] and http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/oidc.html[OpenID Connect v1.0] protocols so you can connect your identity provider systems if they support these protocols.
+Keycloak supports http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/saml.html[SAML v2.0] and http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/oidc.html[OpenID Connect v1.0] protocols. You can connect your identity provider systems if they support these protocols.
 
 [id="managing-users"]
-== Managing Users
+== User management
 
-There’s UI to add, delete and edit users. See: http://www.Keycloak.org/docs/3.2/server_admin/topics/users.html[Keycloak User Management].
+You can add, delete and edit users in the user interface. See: http://www.Keycloak.org/docs/3.2/server_admin/topics/users.html[Keycloak User Management], for more information.
 
 [id="smtp-configurationemail-notifications"]
-== SMTP Configuration/Email Notifications
+== SMTP configuration and email notifications
 
-Eclipse Che does not provide any out-of-the-box SMTP servers. This functionality is enabled in Keycloak itself, `che realm settings > Email`. You will need to provide host, port, username and password, if necessary. Eclipse Che is shipped with the default theme that is used for email templates (registration, email confirmation, password recovery, failed login etc).
+Eclipse Che does not provide any SMTP servers. You enable SMTP servers in Keycloak. `che realm settings > Email`. You specify the host, port, username, and password. Eclipse Che uses the default theme for email templates for registration, email confirmation, password recovery, and failed login.
```

Comment:
* not just 'You'
* use bold for GUI elements
* `che realm settings > Email`. -- put in parentheses or something; it's not very reader-friendly as a separate sentence

Reply:
How about :  Eclipse Che does not provide any pre-configured SMTP servers. This functionality is enabled in Keycloak at *che realm settings > Email*.

Reply:
OK, but use bold for GUI. Also, is it really "che realm", or rather "Che realm"?